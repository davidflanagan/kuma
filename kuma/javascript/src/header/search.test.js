//@flow
import React from 'react';
import { create } from 'react-test-renderer';
import DocumentProvider from '../document-provider.jsx';
import { fakeDocumentData } from '../document-provider.test.js';
import Search from './search.jsx';

test('Search snapshot', () => {
    const search = create(
        <DocumentProvider initialDocumentData={fakeDocumentData}>
            <Search />
        </DocumentProvider>
    );
    expect(search.toJSON()).toMatchSnapshot();
});
