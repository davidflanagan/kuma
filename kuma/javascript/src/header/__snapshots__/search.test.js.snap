// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search snapshot 1`] = `
.emotion-2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 2px 8px;
  border: solid #888 1px;
  border-radius: 8px;
  min-width: 80px;
  -webkit-flex: 2 1;
  -ms-flex: 2 1;
  flex: 2 1;
}

.emotion-0 {
  fill: #333;
  -webkit-flex: 0 0 20px;
  -ms-flex: 0 0 20px;
  flex: 0 0 20px;
}

.emotion-1 {
  border-width: 0 !important;
  font-size: 16px;
  font-weight: bold !important;
  -webkit-flex: 1 1 100px;
  -ms-flex: 1 1 100px;
  flex: 1 1 100px;
  min-width: 60px;
}

<form
  action="/en-US/search"
  className="emotion-2"
  id="nav-main-search"
  method="get"
  role="search"
>
  <svg mock
    className="emotion-0"
  />
  <input
    className="emotion-1"
    id="main-q"
    name="q"
    placeholder="Search MDN"
    type="search"
  />
</form>
`;
